apps/posts/src/views/PostAnalytics/Newsletter/components/NewsLetterRadialChart.tsx

```
@@ -22,14 +23,57 @@ export const NewsletterRadialChart:React.FC<NewsletterRadialChartProps> = ({
+                }
+            };
+        case 'lg':
+            return {
+                barWidth: 46,
+                innerRadiusStart: data.length > 1 ? 72 : 94,
+                fontSize: {
+                    percentage: '1.6rem',
+                    label: '1rem'
+                }
+            };
+        case 'responsive':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)